### ISSUE TITLE
Terrain geometry seem shows unsightly crack in terrain.

### ISSUE THREAD

**Issue by spencerparkin** [AnalyticalGraphicsInc/cesium#7555]
In the screen-shot below, notice the seem in the terrain.  It appears that LOD tiles for the terrain are never loading as I zoom in.  This can be readily found/reproduced by using my web-app indicated in the screen-shot's URl line.

![image](https://user-images.githubusercontent.com/1977523/52538619-0eb70180-2d32-11e9-826e-fab9a0fccf69.png)

**Comment by hpinkos** [comment]
@spencerparkin What terrain provider are you using?

This might be fixed by this PR: #7061

**Comment by spencerparkin** [comment]
Here are my image providers...

`var terrain_image_provider = new Cesium.ArcGisMapServerImageryProvider({url:'//services.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer'});`

`var topo_image_provider = new Cesium.ArcGisMapServerImageryProvider({url:'//services.arcgisonline.com/ArcGIS/rest/services/World_Topo_Map/MapServer'});`

I think you're right.  I just need to wait until that PR is merged into master.  My app is currently deployed with a built copy of Cesium.  I'm going to change that once a new release of it is available that also has my recently merged PR in it.  I just need to hang tight.  Thanks for looking into this.

**Comment by hpinkos** [comment]
@spencerparkin I was wondering what `terrain` provider you were using, not imagery provider.  Are you using `terrainProvider: Cesium.createWorldTerrain()` to add Cesium World Terrain?

**Comment by spencerparkin** [comment]
Oh!  Yes, here's a snippet of my code...

`viewer = new Cesium.Viewer('cesiumContainer', {
        terrainProvider: Cesium.createWorldTerrain({
            requestVertexNormals: true
        }),`

I'm sorry, I'm an idiot.

**Comment by hpinkos** [comment]
No problem!  I wasn't able to reproduce the terrain cracking on my computer, but I'm pretty sure what you're seeing will be fixed by #7061
I'm going to close this issue for now, but feel free to re-open it if you upgrade Cesium after #7061 is merged and you're still seeing problems.  Thanks!
